server/src/fty_srr_worker.cc

```diff
@@ -427,6 +432,60 @@ namespace srr
         return response;
     }
 
+    static std::string zmsg_popstring(zmsg_t *resp){
```

Comment:
we should really put that into common_mlm, since it's present elsewhere (*-rest esp.)